cmd/goal/asset.go

```diff
@@ -603,7 +603,7 @@ var infoAssetCmd = &cobra.Command{
 
 		fmt.Printf("Asset ID:         %d\n", assetID)
 		fmt.Printf("Creator:          %s\n", params.Creator)
-		fmt.Printf("Asset name:       %s\n", params.AssetName)
+		reportInfof("Asset name:       %s\n", params.AssetName)
```

Comment:
I think that `UnitName` would be as susceptible as `AssetName`, wouldn't it ?

Reply:
unit name is limited to 8 characters, would this make any difference?

Reply:
if these contain control characters, it might.